README.md

```diff
@@ -14,7 +14,18 @@ using the crates of the Rust ecosystem.
 These examples are complete, and suitable for copying directly into
 new cargo projects. They are tested and guaranteed to work.
 
+If you'd like to read it locally, [install Rust], and then:
+
+```bash
+$ git clone https://github.com/rust-lang-nursery/rust-cookbook
+$ cd rust-cookbook
+$ cargo install mdbook
```

Comment:
we should point to specific mdbook version to install like in the [rust-book readme](https://github.com/rust-lang/book/blob/master/README.md#requirements) (although I find that their "requirements" heading should be just part of the "building")